ESP32LowLevel/CAN.h

```diff
@@ -0,0 +1,62 @@
+/* 
+   Using https://github.com/collin80/esp32_can
+   which requires https://github.com/collin80/can_common
+   Install both by downloading the ZIP file from GitHub.
+
+*/
+
+#include <esp32_can.h>

```

Comment:
If these are file we need to copy (not installable using the Arduino IDE) I think that we can copy them directly in the codebase in this folder